libminifi/test/TestBase.cpp

```diff
@@ -564,7 +588,7 @@ std::unique_ptr<minifi::Connection> TestPlan::buildFinalConnection(const std::sh
   std::stringstream connection_name;
   connection_name << processor->getUUIDStr() << "-to-" << processor->getUUIDStr();
   auto connection = std::make_unique<minifi::Connection>(flow_repo_, content_repo_, connection_name.str());
-  connection->addRelationship(termination_);
+  connection->addRelationship(termination_.value());
```

Comment:
Shouldn't we check if `termination_` is set to something?

Reply:
in theory `buildFinalConnection` is only called when `termination_` is set, added `gsl_Expects` to verify this precondition